exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Op.kt

```diff
@@ -563,10 +570,13 @@ class exists(
     }
 }
 
+/** Returns an SQL operator that checks if [query] returns at least one row. */
+fun exists(query: AbstractQuery<*>) = Exists(query)
+
 /**
  * Represents an SQL operator that checks if [query] doesn't returns any row.
  */
-class notExists(
+class NotExists(
```

Comment:
Potential breaking change introduced here. If we don't want this, I can revert this change and suppress the `detekt` warning.